x-pack/plugins/endpoint/public/applications/endpoint/models/policy.ts

```diff
@@ -0,0 +1,90 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import { PolicyConfig } from '../types';
+
+/**
+ * Generate a new Policy model.
+ * NOTE: in the near future, this will likely be removed and an API call to EPM will be used to retrieve
+ * the latest from the Endpoint package
+ */
+export const generatePolicy = (): PolicyConfig => {
```

Comment:
this is essentially our defaults for now?

Reply:
@kevinlog  Yes, but this will move to the package registry (as part of the Endpoint package) (here: https://github.com/elastic/package-registry/blob/master/dev/packages/example/endpoint-1.0.0/manifest.yml). Once there, it should not be necessary for us to have this generator here (ok - maybe for UT), and if needed, we can use EPM APIs to retrieve it from the package.